packages/cache-inmemory/src/types/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Shape of a record saved in `in-memory`  cache
+ */
+export type CacheRecord<T> = {
+  data: T
+  /**
+   * Timestamp in milliseconds
+   */
+  expire: number
+}
+
+export type InMemoryCacheModuleOptions = {
+  /**
+   * Time to keep data in cache (in seconds)
+   */
+  defaultTTL?: number
```

Comment:
suggestion(non-blocking): Rename to `ttl`. The default one comes from within the class.